.github/pull_request_template.md

```diff
@@ -0,0 +1,46 @@
+<!--
+First, 🌠 thank you 🌠 for considering a contribution to Apollo!
+
```

Comment:
I think we (I, it's my PR) should add a warning here about the state of the project and our readiness to accept contributions, one thought would be:

```suggestion
 ⚠️ Due to current roadmapping/planning and implementation work being
   worked on by the core team, this project is not yet ready to accept
   external contributions.  With the exception of perhaps _small_ bug
   fixes, please open an issue prior to opening all PRs.  We will open the
   project to contributions as soon as things are moving around a bit less
   and our roadmap is more defined.  Thanks for your understanding!
```

Reply:
Makes sense :+1: